### ISSUE TITLE
XYZ files and missing `pbc` key

### ISSUE THREAD

**Issue by wcwitt** [JuliaMolSim/JuLIP.jl#151]
Here's a test script
```
import Pkg
Pkg.activate(".")
Pkg.add(url="https://github.com/JuliaMolSim/JuLIP.jl", rev="dev-v0.11")
using JuLIP
dataset = JuLIP.read_extxyz("gp_iter6_sparse9k.xml.xyz")
```
which expects to find a Si GAP dataset (can download [here](https://www.repository.cam.ac.uk/handle/1810/317974)).

The script fails with `ERROR: LoadError: KeyError: key "pbc" not found`. I believe it's because there is a (single) config in that dataset without a `pbc` flag.

Note I used the `dev-v0.11` branch because `master` currently fails for an unrelated reason before it gets to this point.

@jameskermode

**Comment by jameskermode** [comment]
Thanks for reporting, I’ll take a look. Probably the best fix is to agree a default for pbc if it’s not present. I propose `pbc=[true, true, true]` should be the default.

**Comment by wcwitt** [comment]
Thanks! Completely happy for you to choose the default, but I note ASE uses False.

**Comment by jameskermode** [comment]
Yes, ASE changed this default from True to False at some point which caused all manner of bugs. @cortner do you have a preference?

**Comment by Sideboard** [comment]
Is there a typo in the title? `pbe` -> `pbc`

**Comment by jameskermode** [comment]
Yes, it should be `pbc`

**Comment by cortner** [comment]
closed by #152
